server/push/router/push_router.js

```diff
@@ -1,15 +1,20 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
 const express = require('express');
-const pushManager = require('./push_manager.js');
+const pushController = require('../controller/push_controller');
 
 const router = express.Router();
+
 router.post('/client', (req, res) => {
```

Comment:
Please don't use req, res. We have to use full name instead of these.
Like request, response.